keps/sig-cluster-lifecycle/kubeadm/4471-cp-join-kubelet-local-apiserver/kep.yaml

```
@@ -0,0 +1,19 @@
+title: Make a control-plane's kubelet point to the local API Server on kubeadm join
+kep-number: 4471
+authors:
+  - "@chrischdi"
+owning-sig: sig-cluster-lifecycle
+participating-sigs:
+  - sig-cluster-lifecycle
+status: provisional
```

```suggestion
status: implementable
```

you can change the status given there are no major objection on our side and this can be implemented in 1.31 if you have time to work on it.